src/advanced_inv.h

```diff
@@ -181,12 +180,22 @@ struct advanced_inv_listitem {
     /**
      * Create a normal item entry.
      * @param an_item The item pointer, stored in @ref it. Must not be null.
+     * @param count The stack size, stored in @ref stacks.
+     * @param index The index, stored in @ref idx.
+     * @param area The source area, stored in @ref area. Must not be AIM_ALL.
+     * @param from_vehicle Is the item from a vehicle cargo space?
+     */
+    advanced_inv_listitem(item *an_item, int index, int count,
+            aim_location area, bool from_vehicle);
+    /**
+     * Create a normal item entry.
+     * @param items The list of item pointers, stored in @ref it.
      * @param index The index, stored in @ref idx.
      * @param count The stack size, stored in @ref stacks.
```

Comment:
Funny thing, this diff looks quite different when excluding whitespace changes, however:

This function doesn't have the count parameter anymore, please remove the documentation of it.